### ISSUE TITLE
DataInspector still shows transformed coordinated for log scale, hline

### ISSUE THREAD

**Issue by KronosTheLate** [MakieOrg/Makie.jl#3526]
This is extremely related to https://github.com/MakieOrg/Makie.jl/issues/3450. That issue was closed, and is indeed generally fixed. But for whatever reason, it is not fixed for a line plotted with hlines:
![image](https://github.com/MakieOrg/Makie.jl/assets/61620837/137a3d62-6330-41f4-87a1-3d8c7673898f)
![image](https://github.com/MakieOrg/Makie.jl/assets/61620837/9ccd808d-d1ae-4cda-a096-0ff739e2851f)

**Comment by ffreyer** [comment]
Yes, because h/vlines handle transform functions explicitly before creating a line plot.

For most recipes DataInspector falls back on primitive plots, i.e. the linesegments plot in plot.plots for h/vlines. This plot has input points in the transformed space, which DataInspector assumes to be data space.
